sites/public/styles/overrides.scss

```diff
@@ -70,7 +70,7 @@ $header-desktop-min-size: 1024px;
 
   --bloom-font-sans: var(--detroit-font-sans);
   --bloom-font-alt-sans: var(--detroit-font-sans);
-  --bloom-color-accent-cool: "#297E73";
+  --bloom-color-accent-cool: #297e73;
```

Comment:
Note: All of these overrides are not taking effect as strings and by removing the quotes, the button focus state issue was resolved. We should remove the quotes from all of these fields eventually, but I thought it would be better to change the others after the release to give us more time to discover any potential side effects.